src/patternfly/components/Login/login.scss

```diff
@@ -0,0 +1,244 @@
+  }
+}
+
+.pf-c-login__header {
+  grid-area: header;
+  margin-bottom: var(--pf-c-login__header--MarginBottom);
+
+  @media (max-width: $pf-global--breakpoint--sm) {
+    padding-right: var(--pf-c-login__header--sm--PaddingRight);
+    padding-left: var(--pf-c-login__header--sm--PaddingLeft);
+  }
+
+  @media (min-width: $pf-global--breakpoint--xl) {
+    margin-top: var(--pf-c-login__header--xl--MarginTop);
+    margin-bottom: var(--pf-c-login__header--xl--MarginBottom);
+  }
+
+  .pf-c-brand {
+    margin-bottom: var(--pf-c-login__header-brand--MarginBottom);
+
+    @media (min-width: $pf-global--breakpoint--xl) {
+      margin-bottom: var(--pf-c-login__header-brand--xl--MarginBottom);
+    }
+  }
+}
+
+.pf-c-login__main {
+  // This component always needs to be light
+  @extend %pf-t-light;
+
+  grid-area: main;
+  display: flex;
+  flex-direction: column;
+  background-color: var(--pf-c-login__main--BackgroundColor);
+
+  // If the first child isn't a header, then we need to put the header's top padding there
+  > :first-child:not(.pf-c-login__main-header) {
+    padding-top: var(--pf-c-login__main-header--PaddingTop);
+  }
+
+  // If the last child isn't the footer, then we need to put the footer's bottom padding there
+  > :last-child:not(.pf-c-login__main-footer) {
+    padding-bottom: var(--pf-c-login__main-footer--PaddingBottom);
+  }
+
+  // This is not mobile first but avoids removing the margin for larger devices
+  @media (max-width: $pf-global--breakpoint--xl) {
+    margin-bottom: var(--pf-c-login__main--MarginBottom);
+  }
+}
+
+.pf-c-login__main-header {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  column-gap: var(--pf-c-login__main-header--ColumnGap);
+  row-gap: var(--pf-c-login__main-header--RowGap);
+  flex: 0 0 auto;
+  align-items: center;
+  padding-top: var(--pf-c-login__main-header--PaddingTop);
+  padding-right: var(--pf-c-login__main-header--PaddingRight);
+  padding-bottom: var(--pf-c-login__main-header--PaddingBottom);
+  padding-left: var(--pf-c-login__main-header--PaddingLeft);
+
+  .pf-c-title {
+    grid-column: -1 / 1;
+  }
+}
+
+.pf-c-login__main-body {
+  flex: 1 1 auto;
+  padding-top: var(--pf-c-login__main-body--PaddingTop);
+  padding-right: var(--pf-c-login__main-body--PaddingRight);
+  padding-bottom: var(--pf-c-login__main-body--PaddingBottom);
+  padding-left: var(--pf-c-login__main-body--PaddingLeft);
+
+  .pf-c-form__group.pf-m-action {
+    display: grid;
+    justify-items: center;
+    grid-gap: var(--pf-c-login__main-body-form-group--m-action--GridGap);
+
+    @media (min-width: $pf-global--breakpoint--md) {
+      grid-template-columns: auto 1fr;
+    }
+
+    .pf-c-button {
+      --pf-c-button--PaddingLeft: var(--pf-c-login__main-body-form-group--m-action__button--PaddingX);
+      --pf-c-button--PaddingRight: var(--pf-c-login__main-body-form-group--m-action__button--PaddingX);
+
+      justify-self: stretch;
+    }
+
+    .pf-c-check {
```

Comment:
`.pf-c-check` should be 24px away from the button according to the [design spec](https://drive.google.com/file/d/1ElopIvkE-RDeqTpJQ2sCXjzXaIbhRafT/view). I'd say just making it fill the grid item container width and relying on grid-gap's 16px to separate the two looks fine though.